Content.Shared/Silicons/StationAi/SharedStationAiSystem.cs

```diff
@@ -451,6 +451,9 @@ private void AttachEye(Entity<StationAiCoreComponent> ent)
         }
 
         _mover.SetRelay(user, ent.Comp.RemoteEntity.Value);
+
+        var eyeName = Loc.GetString("station-ai-eye-name", ("name", MetaData(user).EntityName));
```

Comment:
```suggestion
        var eyeName = Loc.GetString("station-ai-eye-name", ("name", Name(user)));
```
just use the shorthand